module_pcm_bluetooth: missing pthread symbols.
libgstbluetooth: missing symbols from libgstbase-0.10.

--- Makefile.am.orig	2012-01-14 08:13:53.338450039 +0100
+++ Makefile.am	2012-01-14 08:14:22.302766244 +0100
@@ -352,7 +352,7 @@ audio_libasound_module_pcm_bluetooth_la_
 audio_libasound_module_pcm_bluetooth_la_LDFLAGS = -module -avoid-version #-export-symbols-regex [_]*snd_pcm_.*
 audio_libasound_module_pcm_bluetooth_la_LIBADD = sbc/libsbc.la \
 					lib/libbluetooth-private.la @ALSA_LIBS@
-audio_libasound_module_pcm_bluetooth_la_CFLAGS = $(AM_CFLAGS) @ALSA_CFLAGS@
+audio_libasound_module_pcm_bluetooth_la_CFLAGS = $(AM_CFLAGS) @ALSA_CFLAGS@ -pthread
 
 audio_libasound_module_ctl_bluetooth_la_SOURCES = audio/ctl_bluetooth.c \
 					audio/rtp.h audio/ipc.h audio/ipc.c
@@ -386,7 +386,8 @@ audio_libgstbluetooth_la_SOURCES = audio
 audio_libgstbluetooth_la_LDFLAGS = -module -avoid-version
 audio_libgstbluetooth_la_LIBADD = sbc/libsbc.la lib/libbluetooth-private.la \
 						@DBUS_LIBS@ @GSTREAMER_LIBS@ \
-						-lgstaudio-0.10 -lgstrtp-0.10
+						-lgstaudio-0.10 -lgstrtp-0.10 \
+						-lgstbase-0.10
 audio_libgstbluetooth_la_CFLAGS = -fvisibility=hidden -fno-strict-aliasing \
 				$(AM_CFLAGS) @DBUS_CFLAGS@ @GSTREAMER_CFLAGS@
 endif
